indra/newview/lltooldraganddrop.h

```diff
@@ -249,17 +249,20 @@ class LLToolDragAndDrop : public LLTool, public LLSingleton<LLToolDragAndDrop>
                             ESource source,
                             const LLUUID& src_id,
                             bool all_faces,
+                            bool replace_pbr,
```

Comment:
Looks like it should be `remove_pbr` for consistency

Reply:
Agree, missed this, but not worth a separate commit.